package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "superdesk-ui-framework",
-  "version": "3.1.14",
```

Comment:
I have a PR in progress which has v15 as well. Let's merge either this one first, so I can update mine to v16 or I merge first so you update to v16